android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -645,6 +779,98 @@ void createTouchScalar(ImageResolution resolution, DisplayMetrics displayMetrics
         return motionEventList;
     }
 
+    public VideoStreamingParameters getLastCachedStreamingParameters() {
+        return parameters;
+    }
+
+    private List<VideoStreamingCapability> getSupportedCapabilities(
+            List<VideoStreamingRange> ranges,
+            VideoStreamingCapability rootCapability
+    ){
+
+        List<VideoStreamingCapability> validCapabilities = new ArrayList<>();
+        List<VideoStreamingCapability> allCapabilities = rootCapability.getAdditionalVideoStreamingCapabilities();
+        allCapabilities.add(rootCapability);
```

Comment:
@kostyaBoss this will cause an NPE is getAdditionalVideoStreamingCapabilities is null